depmapomics/loading.py

```diff
@@ -891,8 +893,13 @@ def update(samples, stype, bucket, refworkspace, samplesetname=SAMPLESETNAME,
       refsheet_url ([type], optional): [description]. Defaults to REFSHEET_URL.
   """
   # uploading to our bucket (now a new function)
+  # terra.changeToBucket(samples, bucket, name_col=name_col,
+    #                    values=values, filetypes=filetypes, catchdup=True, dryrun=False)
+
+
+  # dryrun is not an argument?
   terra.changeToBucket(samples, bucket, name_col=name_col,
```

Comment:
it is in the new genepy version, this is needed

Reply:
(line below)

Reply:
resolved in new PR